pkgs/by-name/po/powermode-indicator/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  gtkmm3,
+  glibmm,
+  libappindicator,
+}:
+stdenv.mkDerivation {
+  pname = "powermode-indicator";
+  version = "unstable-2024-07-13";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

```suggestion
  version = "0-unstable-2024-07-13";
```

---

fixed